stable/fluent-bit/values.yaml

```diff
@@ -129,14 +129,22 @@ existingConfigMap: ""
 
 rawConfig: |-
   @INCLUDE fluent-bit-service.conf
-  @INCLUDE fluent-bit-input.conf
-  @INCLUDE fluent-bit-filter.conf
```

Comment:
same with the filenames... either we should leave them the same or mention in a migration document saying how to convert.